tests/integration/test_model.py

```diff
@@ -250,3 +296,55 @@ def test_drop_model(c, training_df):
     c.sql("DROP MODEL IF EXISTS my_model")
 
     assert "my_model" not in c.models
+
+
+def test_describe_model(c, training_df):
+    c.sql(
+        """
+        CREATE MODEL my_model1 WITH (
+            model_class = 'sklearn.ensemble.GradientBoostingClassifier',
+            wrap_predict = True,
+            target_column = 'target'
+        ) AS (
+            SELECT x, y, x*y > 0 AS target
+            FROM timeseries
+            LIMIT 100
+        )
+    """
+    )
+    expected_dict = {
```

Comment:
Would it make sense to get those parameters dynamically from an instance of the model (e.g. created with `sklearn.ensemble.GradientBoostingClassifier()` directly)? I am a bit worried that we will need to change this code often whenever there is a new version of `sklearn`.

Reply:
Sure, this is good idea , I will get model from the context and the model parameters as expected data .And compare it with result of the Describe model output

Reply:
Makes sense!